src/components/Filters/helpers.ts

```diff
@@ -0,0 +1,16 @@
+export const validateDefault = (target: HTMLInputElement): boolean | string => {
+  return !!target.validity.patternMismatch;
+};
+
+export const onValidate = (
+  event: React.ChangeEvent<HTMLInputElement>,
+  validate: (target: HTMLInputElement) => boolean | string,
+  patternMessage: string,
+): boolean => {
+  const { target } = event;
+  const error = validate(target);
+  target.setCustomValidity(
+    target.validity.patternMismatch ? patternMessage : '',
```

Comment:
We don't want to display native validation message. It should be removed

Reply:
If removed, the native validation message still displays on "Apply" but with default message "Please match the requested format". IMO it's better to have the same message as the error.